pkgs/applications/misc/octoprint/plugins.nix

```diff
@@ -104,6 +104,25 @@ let
       };
     };
 
+    curaenginelegacy = buildPlugin rec {
```

Comment:
Hm, if it's named `legacy` I guess they finally implemented a plugin for new `curaengine` too.

Reply:
Unfortunately no, as far as I can tell.  They just decided to get rid of the ancient cura version.